app/view-plants/page.tsx

```diff
@@ -1,11 +1,20 @@
 'use client';
 
-import { useEffect, useState } from 'react';
+import { useEffect, useMemo, useState } from 'react';
 import { UUID } from 'crypto';
 import supabase from '@/api/supabase/createClient';
 import { getAllPlants, getPlantById } from '@/api/supabase/queries/plants';
+import FilterDropdownMultiple from '@/components/FilterDropdownMultiple';
 import PlantCard from '@/components/PlantCard';
-import { Plant } from '@/types/schema';
+import SearchBar from '@/components/SearchBar';
+import { DropdownOption, Plant } from '@/types/schema';
```

Comment:
I'm thinking of having Kevin refactor the DropdownOption type into DropdownOption<T> defined below (i.e. making DropdownOption a generic)
```
interface DropdownOption<T> {
   label: string; 
   value: T
}
```
Can you also include that change in schema.d.ts and make sure that all the typing of `DropdownOption[]` is replaced with `DropdownOption<string>[]`?